### PR TITLE
feat: add getRipeWhiteAdminOptions

### PR DESCRIPTION
**Pull request by joaofsl**
| - | - |
| --- | --- |
| Issue | Originated in the discussion of https://github.com/ripe-tech/ripe-util-vue/issues/267 in https://github.com/ripe-tech/ripe-util-vue/pull/281#discussion_r880553385 |

### CODE REVIEW

File: test/env.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
+const assert = require("assert");
+const ripeCommons = require("..");
+
+describe("Env", () => {
```
Comment (joamag) [inline test/env.js:4]: we're not using arrow functions for the unit tests

**Review by joamag — CHANGES_REQUESTED** [review]

File: test/env.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
+const assert = require("assert");
+const ripeCommons = require("..");
+
+describe("Env", () => {
+    describe("#getRipeWhiteAdminOptions()", () => {
```
[the remark this answers is not in the document]
Comment (beneditomauro) [reply test/env.js]: Okay, I will do this in a different PR

File: test/env.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+                baseUrl: "https://ripe-white-admin-ci.platforme.com/"
+            });
+            assert.deepStrictEqual(testEnvOptions, {
+                baseUrl: "https://ripe-white-admin-ci.platforme.com/"
+            });
+        });
+
+        it("should return sbx options for 'now' and 'now-branch' environment", function() {
+            const nowEnvOptions = ripeCommons.getRipeWhiteAdminOptions("now");
+            const nowBrunchEnvOptions = ripeCommons.getRipeWhiteAdminOptions("now-branch");
```
Comment (joao-conde) [inline test/env.js:19]: Im hungry too, but I think this should be `nowBranchEnvOptions`

File: test/env.js [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+        });
+
+        it("should return production options for 'production' environment", function() {
+            const productionEnvOptions = ripeCommons.getRipeWhiteAdminOptions("production");
+            assert.deepStrictEqual(productionEnvOptions, {
+                baseUrl: "https://white-admin.platforme.com/"
+            });
+        });
+
+        it("should return fallback options when environment different than 'ci', 'test', 'stage' and 'production' ", function() {
```
Comment (joao-conde) [inline test/env.js:49]: Write a similar test but when the fallback is something else than {}

File: test/env.js (hunk 1 above)
Comment (joao-conde) [inline test/env.js:4]: Regular function too instead of arrow

File: test/env.js [hunk 5]
```diff
@@ -0,0 +1,54 @@
+const assert = require("assert");
+const ripeCommons = require("..");
```
Comment (joao-conde) [inline test/env.js:2]: new line between 3rd party and our packages

**Review by joao-conde — CHANGES_REQUESTED** [review]

**Review by joao-conde — APPROVED** [review]

**Review by BeeMargarida — APPROVED** [review]

**Review by joamag — APPROVED** [review]

**Review by joamag — APPROVED** [review]

**Pull request merged**